index.bs

```diff
@@ -1142,8 +1152,9 @@ The attestation data encodes contextual bindings made by the <a>authenticator</a
 data has a compact but extensible encoding. This is desired since authenticators can be devices with limited capabilities and
 low power requirements, with much simpler software stacks than the client platform components.
 
-The field `rawData` for this type is a byte array of 45 bytes + length of public key + length of KeyHandle + potentially more 
-extensions. The first bytes before the extensions have a fixed layout as follows: 
+The field `rawData` for this type is a byte array of 17 bytes + length of AAGUID + length of public key + length of Credential
+ID + length of clientDataHash + potentially more extensions. The first bytes before the extensions have a fixed layout as
+follows:
```

Comment:
Overall, other than my prior comment/suggestion, this PR looks good to me  -- thanks for reviewing and implementing those various outstanding issues.